changelog/unreleased/bugfix-media-controls-overflow

```diff
@@ -0,0 +1,6 @@
+Bugfix: Media controls overflow on mobile screens
+
+Media controls overflowed on smaller 9:16 screens because it is absolutely positioned and centered using transform property, its margin caused the transform operation to not calculate the center properly (the latter also affected desktop but was merely visible).
```

Comment:
This is nice technical detail which is great to have as PR description (like you do). For a changelog item however it's a bit much technical detail as those items are usually intended for end users.

An exception are changes that target developers, e.g. changes in the extension framework.